JenkinsfileHW

```diff
@@ -3,28 +3,31 @@ lock(label: 'adgt_test_harness_boards') {
     @Library('sdgtt-lib@adgt-test-harness') _ // Not necessary when we turn on global libraries :)
     def hdlBranch = "NA"
     def linuxBranch = "NA"
-    def bootPartitionBranch = "release"
+    def bootPartitionBranch = "master" // Workaround changes in Artifactory folder structure, master unaffected (HTH-475)
     def firmwareVersion = 'v0.34'
     def bootfile_source = 'artifactory' // options: sftp, artifactory, http, local
     def harness = getGauntlet(hdlBranch, linuxBranch, bootPartitionBranch, firmwareVersion, bootfile_source)
-  
-    //Udpate repos
+
+    //Update repos
     harness.set_env('nebula_repo', 'https://github.com/sdgtt/nebula.git')
     harness.set_env('nebula_branch','dev')
-    harness.set_env('nebula_config_branch','release')
-    harness.set_env('libiio_branch', 'v0.23')
-    harness.set_env('telemetry_repo', 'https://github.com/sdgtt/telemetry.git')
-    harness.set_env('telemetry_branch', 'master')
+    harness.set_env('nebula_config_branch','master')
+    // harness.set_env('libiio_branch', 'v0.23')
+    // harness.set_env('telemetry_repo', 'https://github.com/sdgtt/telemetry.git')
+    // harness.set_env('telemetry_branch', 'master')
```

Comment:
Do we not need these settings anymore?

Reply:
Unlike nebula which has frequent updates, we don't need to update libiio and telemetry at every build since they come pre-installed in the docker container. Setting these values in the Jenkinsfile would download and install them again. I commented the lies to keep a template in case we do need to override the installation in the docker container.